render.py

```diff
@@ -294,12 +295,7 @@ def sjoin(a, sep, b):
     return r
 
 def call_render(workfile, outfile, width, height):
-    # can't use rsvg as the black outline around placeholers is
-    # stored in a way only inkscape can process
-    #rc = subprocess.call(["rsvg-convert", "-w", str(width), "-h", str(height), "-f", "png", "-o", outfile, workfile])
-    rc = subprocess.call(["inkscape", "-z", "--export-png=%s" % outfile, "--export-area-page", "-w", str(width), "-h", str(height), workfile], stdout=dev_null)
-    return rc
-
+    return svg2png(url=workfile, write_to=outfile, output_width=width, output_height=height)
```

Comment:
Nitpick: since this function is only called at one place and now only has a single line, I'd inline it in the calling function.

Reply:
Thought so as well; done now.